apps/main/src/dex/lib/curvejs.ts

```diff
@@ -1310,12 +1311,21 @@ const poolWithdraw = {
   },
 }
 
+/**
+ * TODO: Work around for getPoolList being called twice from UserSlice and DashboardSlice
```

Comment:
Prob temporary until we refactor to use tanstack instead of stores

Reply:
indeed #696